packages/app/src/runner/SpecRunnerHeader.vue

```diff
@@ -1,54 +1,107 @@
+        }"
       >
-        <div
-          class="rounded-md flex shadow-md mx-2 px-4 url"
-          :class="{
-            'bg-yellow-50': autStore.isLoadingUrl,
-            'bg-white': !autStore.isLoadingUrl,
-          }"
+        <Button
+          data-cy="header-selector"
+          :disabled="isDisabled"
+          class="rounded-none border-r-1px border-gray-100 mr-12px"
+          variant="text"
+          @click="togglePlayground"
         >
-          <div>
-            {{ autStore.url }}
-          </div>
+          <i-cy-crosshairs_x16 class="icon-dark-gray-600" />
+        </Button>
+        <div class="mr-12px max-w-100% grid text-gray-600 items-center truncate">
+          {{ autStore.url }}
         </div>
-
-        <div>Loading URL: {{ autStore.isLoadingUrl }}</div>
       </div>
-
-      <Select
-        :model-value="browser"
+      <div
+        v-else
+        class="flex-grow"
+      >
+        <!-- spacer -->
+      </div>
+      <SpecRunnerDropdown
+        v-if="selectedBrowser?.displayName"
         data-cy="select-browser"
-        :options="browsers"
-        item-value="displayName"
-        @update:model-value="changeBrowser"
-      />
+      >
+        <template #heading>
+          <img
+            class="min-w-16px w-16px"
+            :src="allBrowsersIcons[selectedBrowser.displayName]"
+          > {{ selectedBrowser.displayName }} {{ selectedBrowser.majorVersion }}
+        </template>
+
+        <template #default>
+          <div class="max-h-50vh overflow-auto">
+            <VerticalBrowserListItems
+              :gql="props.gql"
+            />
+          </div>
+        </template>
+      </SpecRunnerDropdown>
+      <SpecRunnerDropdown
+        variant="panel"
+        data-cy="viewport"
+      >
+        <template #heading>
+          <i-cy-ruler_x16 class="icon-dark-gray-500 icon-light-gray-400" />
+          <span class="whitespace-nowrap">{{ autStore.viewportWidth }}x{{ autStore.viewportHeight }}</span>
+          <span
+            v-if="displayScale"
+            class="-mr-6px text-gray-500"
+          > ({{ displayScale }})</span>
+        </template>
+        <template #default>
+          <div class="max-h-50vw p-16px text-gray-600 leading-24px w-400px overflow-auto">
+            <p class="mb-16px">
+              The
+              <strong>viewport</strong> determines the width and height of your application.
+              By default the viewport will be
+              <strong>{{ autStore.defaultViewportWidth }}px</strong> by
+              <strong>{{ autStore.defaultViewportHeight }}px</strong> for {{ props.gql.currentTestingType === "e2e" ? 'End-to-end' : 'Component' }}
+              Testing unless specified by a <InlineCodeFragment>cy.viewport</InlineCodeFragment> command.
+            </p>
+            <p class="mb-16px">
+              Additionally you can override the default viewport dimensions by specifying these values in your config file:
```

Comment:
I'd add `Additionally,` (comma). Is there a reason this isn't using `t('...')`?

Reply:
Avoiding translation because the content is not confirmed, and I didn't want to do the interpolation. It's just placeholder content until we get confirmation.